appserver/pom.xml

```diff
@@ -304,7 +304,7 @@
              <dependency>
                 <groupId>com.sun.messaging.mq</groupId>
                 <artifactId>imqjmx</artifactId>
-                <version>4.3</version>
```

Comment:
Haven't compared them, but there is 
https://repo.maven.apache.org/maven2/org/glassfish/mq/imqjmx/5.1/
(with even higher version).

Reply:
Tbh, I don't know what it exactly is, or why it's there. It wasn't updated for a long time, maybe exactly because nobody seems to know what it is 😬

Would be good to investigate what that org/glassfish imqjmx is, why it's there, where it comes from, and why GF never used it.